lib/cocoapods/installer/xcode/pods_project_generator/pod_target_integrator.rb

```diff
@@ -83,22 +91,34 @@ def add_copy_resources_script_phase(native_target, test_spec)
           #
           def add_embed_frameworks_script_phase(native_target, test_spec)
             script_path = "${PODS_ROOT}/#{target.embed_frameworks_script_path_for_test_spec(test_spec).relative_path_from(target.sandbox.root)}"
-            input_paths = []
-            output_paths = []
+
+            input_paths_by_config = {}
+            output_paths_by_config = {}
             unless installation_options.disable_input_output_paths?
               framework_paths = target.dependent_targets_for_test_spec(test_spec).flat_map do |dependent_target|
                 spec_paths_to_include = dependent_target.non_test_specs.map(&:name)
                 spec_paths_to_include << test_spec.name if dependent_target == target
-                dependent_target.framework_paths.values_at(*spec_paths_to_include).flatten.compact.uniq
-              end
+                dependent_target.framework_paths.values_at(*spec_paths_to_include).flatten.compact
+              end.uniq
+
               unless framework_paths.empty?
-                input_paths = [script_path, *framework_paths.flat_map { |fw| [fw.source_path, fw.dsym_path] }.compact]
-                output_paths = UserProjectIntegrator::TargetIntegrator.framework_output_paths(framework_paths)
+                input_file_list_path = target.embed_frameworks_script_input_files_path_for_test_spec(test_spec)
+                input_file_list_relative_path = "${PODS_ROOT}/#{input_file_list_path.relative_path_from(target.sandbox.root)}"
```

Comment:
can this use the logic in `AggregateTarget#embed_frameworks_script_input_files_relative_path` somehow? it feels like this logic should be grouped together somewhere. maybe `AggregateTarget` should call out to `PodTargetIntegrator` to get the path

Reply:
not really, because this is for a test spec. this is how we also did the path for the test spec's embed frameworks script

Reply:
ah gotcha, totally missed that this is for a test spec